frontend/src/admin/common/tables/ColumnHeader.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+import SortButton from './SortButton';
+
+const TH = styled.th`
+padding: 0;
+&:last-child {
+    width: 100px;
+}
+`;
+
+const ColumnHeader = ({ label, type = 'edit', onClick }) =>
+  <TH className={`${type}`} onClick={onClick}>
```

Comment:
To display the relevant chevron - which had stopped happening at some point. Should be fixed now.